client/src/Event/EventApi.ts

```diff
@@ -85,7 +87,7 @@ export interface TsrEvent {
     organizations: Organization[];
     startDate: string;
     endDate: string;
-    eventType?: EventType;
+    eventType: EventType[];
```

Comment:
I don’t think we want multiple types for events. At least that wasn’t existing behavior. We can talk about it to see if this is the direction we want to go with it.

Reply:
Component has been refactored to be a single event type select.